data/core/help.cfg

```diff
@@ -268,6 +268,15 @@ Each unit has a certain number of movement points which are used up when moving
 Another thing to keep in mind while moving is <italic>text='zones of control'</italic>. Each unit — except for level 0 units — generates a zone of control in the hexes immediately surrounding it, and any enemy unit entering those hexes immediately ends its movement. Learning how to use zones of control to your advantage is an important part of Wesnoth, as only <ref>dst='ability_skirmisherskirmisher' text='skirmishers'</ref> can ignore zones of control." + _ "
 
 To see where the enemy can move to during their next turn, press Ctrl-v or Cmd-v. Ctrl-b or Cmd-b shows where the enemy could move, if your units were not on the map to block their progress."
+    [/topic]
+
+	# wmllint: markcheck on
+
+    # wmllint: markcheck off
+    [topic]
+        id=vision
+        title= _ "Vision"
+        text=_ "Vision determines how far a unit can see on the map. Normally, a unit’s vision range is calculated as its movement range plus one hex. Certain units with unique vision capabilities deviate from this rule and have vision cost independent of movement range. Vision is also impacted by certain map features like forests or hills which can obstruct a unit's line of sight."
```

Comment:
Saying "line of sight" is misleading, because the Wesnoth definition of vision can bend round corners just like movement can.

Reply:
Yeah, I think the last sentence is not great. There are no separate terrain features that influence vision (or movement). For a specific unit type certain terrain has a certain vision cost value (that defaults to the movement cost) and that's it. There is no line of sight calculation or so.

I think that sentence can just be left out.

Reply:
Okay, that is more than fair. Should I just remove the last sentence or is there something else that should be changed?

Reply:
I think removing it is enough.

Sorry that it took so long to return to this review.